### PR TITLE
Konyang: Point Verdant shops and sewers

### PR DESCRIPTION
**Pull request by DreamySkrell**
changes:
  - tweak: "Point Verdant: shops and sewers."

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

File: html/changelogs/DreamySkrell-konyang-stuff.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - tweak: "Point Verdant: stuff."
```
Comment (SleepyGemmy) [inline html/changelogs/DreamySkrell-konyang-stuff.yml:41]: describe what you are adding.

File: maps/away/away_site/konyang/point_verdant/point_verdant-1.dmm
Comment (SleepyGemmy) [inline maps/away/away_site/konyang/point_verdant/point_verdant-1.dmm:1]: remove the dirt from the sewer water.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
the PR has no description.

File: maps/away/away_site/konyang/point_verdant/point_verdant-1.dmm
Comment (DreamySkrell) [reply maps/away/away_site/konyang/point_verdant/point_verdant-1.dmm:1]: done

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**